docs/modules/ROOT/pages/includes/attributes.adoc

```diff
@@ -1,4 +1,4 @@
-:quarkus-version: 2.5.1.Final
+:quarkus-version: 2.6.0.Final
```

Comment:
Can we get this info from Maven?

Reply:
It's coming from Maven during the build. The problem is that you need it defined in attributes for Antora to pick them up.

It's fixed when releasing anyway as the release process updates it.

Reply:
The problem is that this needs to be manually updated when dependabot opens a PR updating quarkus

Reply:
Yes, I understand that. Be my guest for finding a better solution :). We could trigger an additional commit automatically but given it's naturally fixed on release anyway, I don't see it as a major problem and I decided to spend my time elsewhere as the situation is good enough (TM) for me :).

Reply:
Sure, I'll play around with maven-resources-plugin and the filtering mechanism here, but I agree this is good enough for now